Gemfile

```diff
@@ -5,20 +5,20 @@ gem "sinatra", "1.3.4"
 gem "rake", "~> 10.5"
 gem "rack", "~> 1.6"
 gem "rest-client", "1.8.0"
-gem "logging", "1.8.1"
-gem 'nokogiri', "1.5.5"
+gem "logging", "2.1.0"
+gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
-gem "slop", "3.4.5"
-gem "sidekiq", "2.13.0"
+gem "slop", "4.3.0"
+gem "sidekiq", "< 3"
```

Comment:
As you can see in the `Gemfile.lock` (https://github.com/alphagov/rummager/blob/upgrade-gems-part-1/Gemfile.lock#L140) this hasn't actually updated the sidekiq gem because the current version `2.13` still satisfies this new constraint (`< 3`).